testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-delta-lake-databricks-snappy/delta.properties

```diff
@@ -0,0 +1,9 @@
+connector.name=delta_lake
+hive.metastore=glue
+hive.metastore.glue.region=${ENV:AWS_REGION}
+# We need to give access to bucket owner (the AWS account integrated with Databricks), otherwise files won't be readable from Databricks
+hive.s3.upload-acl-type=BUCKET_OWNER_FULL_CONTROL
+delta.enable-non-concurrent-writes=true
+delta.hive-catalog-name=hive
+# Databricks does not support ZSTD until version 10.4
```

Comment:
Given that Databricks 7.3 and 9.1 LTS are still not EOL, I wonder if it is safe to switch to ZSTD by default.